client/jetpack-connect/style.scss

```diff
@@ -686,3 +686,21 @@
 		padding-right: 319px;
 	}
 }
+
+.jetpack-connect__plan-info {
+	text-align: center;
+	max-width: 500px;
```

Comment:
The 500px width here are a little less visually than on the mockup by @joanrho. I'm good with it this way, but let's get her to approve it too 😉

Reply:
@AnnaMag @tyxla I'm good with this too. We can iterate later and focus on first getting some data on this first version.